change-notes/1.19/analysis-cpp.md

```diff
@@ -16,6 +16,7 @@
 |----------------------------|------------------------|------------------------------------------------------------------|
 | Resource not released in destructor | Fewer false positive results | Placement new is now excluded from the query. |
 | Missing return statement (`cpp/missing-return`) | Visible by default | The precision of this query has been increased from 'medium' to 'high', which makes it visible by default in LGTM. It was 'medium' in release 1.17 and 1.18 because it had false positives due to an extractor bug that was fixed in 1.18. |
+| Missing return statement | Fewer false positive results | The query is now produces correct results when a function returns a template dependent type. |
```

Comment:
```suggestion
| Missing return statement | Fewer false positive results | The query is now produces correct results when a function returns a template-dependent type. |
```
The Standard just says "dependent type", but there's usually some context that makes the "template" part clear. I think hyphenating "template-dependent" makes it more clear that it's a type dependent on the template, not a dependent type that is a template.

Reply:
Thanks.

Reply:
Thanks.

Reply:
Thanks.